css/jquery.uls.css

```diff
@@ -76,7 +76,7 @@
 
 /* There are two input boxes. This class applies to both of them */
 .uls-filterinput {
-	font-size: 16px;
+	font-size: 1.2em;
```

Comment:
That should be `1.143em` for 16px equiv. I use http://pxtoem.com/ for manual conversions as helper